spec/support/aws/config-file.csv

```diff
@@ -1,2 +1,2 @@
 User Name,Access Key Id,Secret Access Key
-test,12345,abcde
+test,67890,fghij
```

Comment:
I changed this just to assert that the data that is read in is different from the data that resides in the `ini-file.ini`. Basically to give a bit more confidence that the test is asserting the right thing.